charts/connect/ci/with-operator-values.yaml

```diff
@@ -2,3 +2,6 @@ operator:
   create: true
   token:
     name: op-operator-connect-token
+
+connect:
+  replicas: 1
```

Comment:
Why do we need this spec now? Doesn't it have a default to 1?

Reply:
It was an attempt to CI. Agree, we need to revert it.